terragrunt/.checkov.yml

```diff
@@ -1 +1,12 @@
 skip-check:
+  - CKV2_AWS_5  # Security Groups: False positive, this is attached to the lambda 
+  - CKV2_AWS_31 # WAF: WAF logging is not required
+  - CKV2_AWS_32 # CloudFront: false-positive, strict header security policy is being applied 
+  - CKV2_AWS_38 # Route53: DNS Secruity Extensions (DNSSEC) signing not required 
+  - CKV2_AWS_39 # Route53: Domain Name System (DNS) query logging not required 
```

Comment:
Just a minor correction here - this is also a false-positive since we've got DNS query logging, but checkov can't see it in the module we're using.

Reply:
Corrected - thank you so much for the clarification.